src/Components/Auth/ForgetPassword/ChangePassword/ChangePassword.scss

```diff
@@ -0,0 +1,227 @@
+$blue-text-color: #2a96cc;
+$oauth-text-color: hsla(120, 100%, 0%, 0.302);
+$box-shadow-color: hsla(0, 0%, 0%, 0.102);
+$button-background-color: #2e95c9;
+$button-text-color: #f7f7f7;
+$input-text-color: hsla(0, 0%, 0%, 0.561);
+$input-bottom-border-color: hsla(0, 0%, 0%, 0.329);
+$font-family: "Roboto",
+sans-serif;
+
+.update-password {
+  align-items: center;
+  height: 100vh;
+  margin-left: 37%;
+  position: absolute;
+  right: 0;
+  width: 60%;
+}
+
+.updatepassword {
```

Comment:
Find better Class name to make a difference between **.update-password** and **.updatepassword**.